baseplate/core.py

```diff
@@ -317,19 +324,59 @@ def __exit__(self, exc_type, value, traceback):
         else:
             self.finish()
 
+    def make_child(self, name, local=False, component_name=None):
+        """Return a child Span whose parent is this Span.
 
-class ServerSpan(Span):
-    """A server span represents a request this server is handling.
+        The child span can either be a local span representing an in-request operation
+        or a span representing an outbound service call.
 
-    The server span is available on the :term:`context object` during requests
-    as the ``trace`` attribute.
+        In a server, a local span represents the time spent within a
+        local component performing an operation or set of operations.
+        The local component is some grouping of business logic,
+        which is then split up into operations which could each be wrapped
+        in local spans.
 
-    """
+        """
+        raise NotImplementedError
 
-    def make_child(self, name):
-        """Return a child span representing an outbound service call."""
+
+class LocalSpan(Span):
+    # pylint: disable=invalid-name
```

Comment:
Hmm, I don't think there's one. This was copypasta'd from the definition in `Span`, but I just removed and ran linting and there wasn't complaining. Removing.